docs/gettingstarted.md

```diff
@@ -1,50 +1,66 @@
 # Getting Started
 
-The Rancher CCG gMSA Plugin feature comprises two components, the plugin installer (which installs the CCG plugin), and the account provider (which the plugin uses to obtain login credentials for a domain). Installing both Helm charts is required in order for a cluster to support domainless GMSA.  
+Installing Rancher CCG gMSA Plugin requires the installation of two Helm charts onto your Windows cluster:
 
-## Prerequisites 
+1. [`rancher-gmsa-plugin-installer`](../charts/rancher-gmsa-plugin-installer): install the CCG Plugin onto each Windows host in your cluster
+2. [`rancher-gmsa-account-provider`](../charts/rancher-gmsa-account-provider): serves as a local proxy for the CCG Plugin to grab a Secret from the Kubernetes cluster
 
-+ Kubernetes v1.24+, containerD 1.7+
-+ Nodes running `Windows Server 2019`, `Windows Server 2022`, `Windows Server Core 2019`, or `Windows Server Core 2022 
-`+ Upstream gMSA Web-hook chart installed ([Official Repo](https://github.com/kubernetes-sigs/windows-gmsa), [Rancher Specific Chart](https://github.com/rancher/charts/tree/release-v2.7/charts/rancher-windows-gmsa))\
-  + **Note:** Currently, the gMSA Web-Hook chart offered by Rancher **does not contain the required changes to support non-domain joined nodes**. This chart will be updated soon, in the meantime you should use the following repository `harrisonwaffel/charts` and test against the `update-gmsa` branch (this can be done by configuring a new app repository in the Rancher UI).
-+ An Active Directory instance which is reachable by worker nodes
-+ One or more Active Directory gMSA accounts 
+Once installed, you may also want to install [`rancher-windows-gmsa`](../charts/rancher-windows-gmsa) (and `rancher-windows-gmsa-crd`), which adds [admission webhooks](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/) on Pods to fill out gMSA credentials based on configuration stored in a `GMSACredentialSpec`. This will allow your Pods to utilize the gMSAs that you registered in Active Directory via an "impersonation" user credential that your Rancher CCG gMAA Plugin has access to.
+
+## Prerequisites
+
+- A Kubernetes cluster with Windows nodes
+  - Must be 1.24+
+  - Must come with [`cert-manager`](https://cert-manager.io/docs/installation) and its CRDs installed
+  - Each node must be running [`containerd`](https://containerd.io) 1.7+ as the runtime
+  - All Windows hosts should have the same OS version. Only `Windows Server 2019` or `Windows Server 2022` are supported.
```

Comment:
I'm not sure this should be a prerequisite as it isn't a hard requirement at a technical level, this solution will work if even if a cluster has a mix of 2019 and 2022 nodes

Reply:
> All Windows hosts should must be running `Windows Server 2019` or `Windows Server 2022`, no other versions are supported.